_posts/2021-08-15-openttd-12-0-beta1.md

```diff
@@ -0,0 +1,47 @@
+---
+title: OpenTTD 12.0-beta1
+author: TrueBrain
+---
+
+Are you dreaming?
+No, you are not!
+
+Today I have the pleasure to announce to you that we just released the first beta of our 12.X release series!
+
+With our release on Steam not even six months ago we got some excellent feedback from the community on what to improve next.
+One of the things that stood out: playing together is not easy for many players.
+
+So, in 12.0 we address this problem.
+Playing together is as easy as one of you starting a server, marking it either "Public" or "Invite-Only", and sharing the newly introduced invite-code with your friends.
+They can join your server, just like that.
+No setup needed on your side, no port-forwarding, nothing.
+
+Sadly, to set expectations, no integration with Steam itself has been added.
+Integrating with Steam as GPL-licenses software has challenges, which we haven't resolved yet.
+But we hope you agree that this new multiplayer experience is the next best thing.
+
+Next to this excellent new feature, a few other new things we introduced are:
+* Brand new "Online Players" window to easier see who is in what company, etc.
+* Maximum number of NewGRFs you can use in one game has been raised to 255.
+* Our official Windows builds are now signed.
+* Define refittability of default vehicles using cargo classes.
```

Comment:
`Default vehicles can now be refitted to multiple cargos when used with industry NewGRFs.`

or

`Default vehicles are now compatible with almost all industry NewGRFs`